### DISCUSSION TITLE
Reporting Security Issues with WallConnect Client Libraries

### DISCUSSION THREAD

**Discussion by sei-vsarvepalli** [WalletConnect/walletconnect-monorepo#3963] [Developer Support]
We have been trying to reach some one from WalletConnect to reporting potential security issues and/or vulnerabilities in the library. Is there a best way to reach your staff? We have tried sending emails from support@walletconnect.com and contact@walletconnect.com but no responses. This is being sent from CERT/CC vulnerability coordination platform https://kb.cert.org/

**Comment by chris13524** [comment]
Please refer to our [security.txt](https://walletconnect.com/.well-known/security.txt) for reporting security issues 🙏

**Reply by sei-vsarvepalli** [reply]
Thanks very much. On looking at our past communications, looks like a notification was also sent to security@walletconnect.com with details to collect the disclosure information.  Did not get a response to that too.
